### ISSUE TITLE
[Bug]: Result error of func lpad when the second para is transferedable char

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#2315]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.4.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

mysql> select * from t1;
+------+------------+----------------------+
| id   | c          | vc                   |
+------+------------+----------------------+
|    1 | Daffy      | Aducklife            |
|    1 | Daffy      | Aducklife            |
|    2 | Bugs       | Arabbitlife          |
|    3 | Cowboy     | Lifeontherange       |
|    4 | Anonymous  | Wannabuythisbook?    |
|    5 | BestSeller | OneHeckuvabook       |
|    5 | BestSeller | OneHeckuvabook       |
|    6 | EveryoneBu | Thisverybook         |
|    7 | SanFran    | Itisasanfranlifetyle |
|    8 | BerkAuthor | Cool.Berkly.the.book |
|    9 | NULL       | NULL                 |
|   10 | 北京市     | 中关村               |
|   10 |            |                      |
+------+------------+----------------------+
13 rows in set (0.02 sec)

mysql> select lpad(c,'1','*') from t1;
ERROR 2027 (HY000): Malformed packet

### Expected Behavior

mysql> select lpad(c,'1','*') from t1;
+-----------------+
| lpad(c,'1','*') |
+-----------------+
| D               |
| D               |
| B               |
| C               |
| A               |
| B               |
| B               |
| E               |
| S               |
| B               |
| NULL            |
| 北              |
| *               |
+-----------------+
13 rows in set (0.01 sec)

**Comment by broccoliSpicy** [comment]
I think I can fix this in v0.5

**Comment by fengttt** [comment]
Done and verified -- though not sure if utf charset/encoding is correct or well defined.   But that is another issue.
